netdata-installer.sh

```diff
@@ -69,18 +73,20 @@ CFLAGS="${CFLAGS--O2}"
 [ "z${CFLAGS}" = "z-O3" ] && CFLAGS="-O2"
 
 # keep a log of this command
-printf "\n# " >>netdata-installer.log
-date >>netdata-installer.log
-printf "CFLAGS=\"%s\" " "${CFLAGS}" >>netdata-installer.log
-printf "%q " "$0" "${@}" >>netdata-installer.log
-printf "\n" >>netdata-installer.log
+{
+    printf "\\n# ";
+    date;
+    printf "CFLAGS=\"%s\" " "${CFLAGS}";
+    printf "%q " "$0" "${@}";
+    printf "\\n" ;
+}>>${run_logfile}
 
 REINSTALL_PWD="${PWD}"
-REINSTALL_COMMAND="$(printf "%q " "$0" "${@}"; printf "\n")"
+REINSTALL_COMMAND="$(printf "%q " "$0" "${@}"; printf "\\n")"
```

Comment:
You have added an extra backslash to `\n`. We just need a new line to be printed here, since `printf` does not print new lines.

Reply:
This will still be priting a newline. https://github.com/koalaman/shellcheck/wiki/SC1117

Reply:
This fix is "retired".... so they got it back.

Reply:
Looks like I am just using shellcheck 0.4.7 in which this is not retired.